src/Riok.Mapperly/Configuration/NestedPropertyMappingConfiguration.cs

```diff
@@ -0,0 +1,3 @@
+namespace Riok.Mapperly.Configuration;
+
+public record NestedPropertyMappingConfiguration(StringMemberPath Source);
```

Comment:
I'd rename to `MapNestedPropertiesMappingConfiguration` to match other namings and attribute name.